### ISSUE TITLE
old mapsource does not understand <bounds  ...></bounds>

### ISSUE THREAD

**Issue by wrohdewald** [tkrajina/gpxpy#186]
it just refuses to load the gpx file, no reason given.

Mapsource only understands <bounds ... />

I believe this should always be correct syntax, so this change should not do any harm. But it makes the output a bit shorter.

**Comment by tkrajina** [comment]
OK, try the following
```
make analyze-xsd
```
it will download the xsd files in `xsd/`.

And then install xmllib, download your gpx file and try:
```
xmllint --noout --schema xsd/gpx1.0.xsd test_files/Mojstrovka.gpx
```
or
```
xmllint --noout --schema xsd/gpx1.1.xsd test_files/Mojstrovka.gpx
```
(depending on the gpx version).

**Comment by wrohdewald** [comment]
How would that help?

Let me rephrase my problem:

AFAIK, both those variants are legal:

`<bounds minlat="53.5" maxlat="53.6" minlon="9.5" maxlon="9.6" ></bounds>`
`<bounds minlat="53.5" maxlat="53.6" minlon="9.5" maxlon="9.6" />`

Gpxpy.to_xml() creates the first variant, Mapsource only understands the second variant. I know a lot of people still use Mapsource.

**Comment by tkrajina** [comment]
You said "I believe", I wanted to be sure that the problem is not the generated GPX. It's quite possible that gpxpy generates an invalid XML file. And, mapsource being a popular app -- I thought maybe the problem *is* in the GPX file. If that's not the case, I agree with you, but it's also not a bug in another application. And here, it's not a bug, it's a feature request.

Anyway, I did try to fix this in `empty-tags`, but some tests are still failing. I'll probably finish it after I merge the current `dev` branch in master and release the next version.

**Comment by tkrajina** [comment]
A fix for this is now in the latest `dev` branch (will be included in the next version).
